src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -2722,8 +2722,8 @@ private function pushChangesToStagingArea($id) {
     $commit = $api->getHeadCommit();
     $prefix = idx($staging, 'prefix', 'phabricator');
 
-    $base_tag = "refs/tags/{$prefix}/base/{$id}";
-    $diff_tag = "refs/tags/{$prefix}/diff/{$id}";
+    $base_tag = "refs/heads/{$prefix}/base/{$id}";
```

Comment:
Per the discussion: remove `heads/` from the ref names for both base & diff

Reply:
Done. I haven't confirmed if this is safe (can $prefix be set to something bad?).